lib/vrl/compiler/src/lib.rs

```diff
@@ -14,7 +14,7 @@
     clippy::cast_precision_loss, // allowed in initial deny commit
     clippy::cast_sign_loss, // allowed in initial deny commit
     clippy::if_not_else, // allowed in initial deny commit
-    clippy::let_underscore_drop, // allowed in initial deny commit
+    let_underscore_drop, // allowed in initial deny commit
```

Comment:
**nit**: should this get re-alphabetized now that it doesn't start with `clippy`?